app/views/works/_form.html.erb

```diff
@@ -0,0 +1,33 @@
+<h3><%=page_title %></h3>
+<% if @work.errors.any? %>
+  <ul class="errors">
+    <% @work.errors.each do |column, message| %>
+    <li>
+      <strong><%= field.capitalize %></strong> <%= message %>
```

Comment:
This is crashing the app when I try to create a work without filling in any of the fields in the form.  

`field` should be `column`

Lastly you should do this with `flash` notices.